gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -1346,7 +1364,7 @@ gst_tensor_src_iio_start (GstBaseSrc * src)
 
   /** get all the channels that exist and then set enable on them */
   dirname = g_build_filename (self->device.base_dir, CHANNELS, NULL);
-  guint num_channels_enabled =
+  gint num_channels_enabled =
```

Comment:
After merging https://github.com/nnsuite/nnstreamer/pull/1213, this declaration may not be allowed.

Reply:
done.